.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  docker:
+    environment: deployment
+    runs-on: ubuntu-latest
+    steps:
+      - name: QEMU setup
+        uses: docker/setup-qemu-action@v1
+      - name: Docker Buildx Setup
+        uses: docker/setup-buildx-action@v1
+      - name: Docker Login
+        uses: docker/login-action@v1
+        with:
+          username: target
```

Comment:
I think the username we need here is actually `svdrone2` but it might be better to store that as a secret with the token.